src/Examples/UnitModels/Reactors/msr_equilibrium_constant.py

```diff
@@ -0,0 +1,184 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Methods for calculating equilibrium constants
+"""
+from pyomo.environ import exp, log, Var, units as pyunits, value
+
+from idaes.models.properties.modular_properties.base.utility import \
+    ConcentrationForm
+from idaes.models.properties.modular_properties.reactions.dh_rxn import \
+    constant_dh_rxn
+from idaes.core.util.misc import set_param_from_config
+from idaes.core.util.constants import Constants as c
+from idaes.core.util.exceptions import BurntToast, ConfigurationError
+
+
+# -----------------------------------------------------------------------------
+# custom class from empirical correlations, modified from van_t_hoff (line 97)
+# Source: Int. J. Hydrogen Energy, 42 (2017), pp. 2889-2903
+class empirical_1():  # steam methane reforming, reaction 1
+
+    @staticmethod
+    def build_parameters(rblock, config):
+        parent = rblock.parent_block()
+        units = parent.get_metadata().derived_units
+
+        c_form = config.concentration_form
+        if c_form is None:
+            raise ConfigurationError(
+                "{} concentration_form configuration argument was not set. "
+                "Please ensure that this argument is included in your "
+                "configuration dict.".format(rblock.name))
+        elif (c_form == ConcentrationForm.moleFraction or
+              c_form == ConcentrationForm.massFraction or
+              c_form == ConcentrationForm.activity):
+            e_units = pyunits.dimensionless
```

Comment:
Do we need all this code supporting different equilibrium forms if we're using this for a single example?

Also, I think the equilibrium relationships you have should be able to put into a form that uses the existing `van_t_hoff` method. Walking the user through this process might actually be good to include in the notebook.